src/base/_defaults.scss

```diff
@@ -145,10 +145,13 @@ pre {
 /**
  * When certain monospace inline elements are not within a preformatted block or
  * a link, we offset their color from surrounding content.
+ *
+ * We're disabling a selector rule here because the alternative is pretty
+ * ridiculously verbose. We should consider revising our selector rules.
  */
 
-*:not(pre):not(a) > code,
-*:not(pre):not(a) > kbd {
+/* stylelint-disable-next-line selector-not-notation */
+:not(pre, pre *, a) > :is(code, kbd) {
```

Comment:
I could not figure out how to get our linter to be okay with `:not(pre *)`, so I decided to keep the less verbose selector and simplify everything down to one line.